src/tests/Interop/CMakeLists.txt

```diff
@@ -110,4 +110,5 @@ if(CLR_CMAKE_TARGET_APPLE)
     add_subdirectory(Swift/SwiftInvalidCallConv)
     add_subdirectory(Swift/SwiftRetAbiStress)
     add_subdirectory(Swift/SwiftSelfContext)
+    add_subdirectory(Swift/SwiftStructLowering)
```

Comment:
I'm a bit worried that the naming here would be a bit confusing because we already test Swift struct lowering in `SwiftAbiStress` and `SwiftRetAbiStress` test scenarios. I would propose either changing it to something `InlineArray` specific or include the newly introduced test in the `SwiftAbiStress` and `SwiftRetAbiStress` tests.

Reply:
I guess this boils down to a question whether we need stress tests for `InlineArray`. My thinking is that not necessarily as they would mostly duplicate the existing stress tests. But I'm curious to learn what others think about it.

Reply:
I agree with that and in my opinion, I wouldn't create additional stress tests for `InlineArray` structs.